Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/CrashesContentPage.xaml.cs

```diff
@@ -251,5 +251,10 @@ private void TrackException(Exception e)
             RefreshPropCount();
             Crashes.TrackError(e, properties);
         }
+
+        void ClearCrashUserConfirmation(object sender, EventArgs e)
+        {
+            DependencyService.Get<IClearCrashClick>().ClearCrashButton();
```

Comment:
It does not work on UWP

![Capture](https://user-images.githubusercontent.com/623672/61663789-3ff31080-ac86-11e9-8e3d-4f4cd6782509.PNG)

Reply:
Also demo UWP has the crash preview nugets so you can do demo as well after fixing forms.puppet.

Reply:
Add clear crash user confirmation for Demo.UWP in a new commit, and forms.puppet is ok. please review.

Reply:
forms.puppet is not ok: https://github.com/microsoft/appcenter-sdk-dotnet/pull/1030#discussion_r306022636

This needs to be fixed @Chaoyun1234

Reply:
@guperrot Sorry, I got it wrong, I have fixed forms.puppet.

Reply:
Only a blank line was added, still crashing.

I looked at and the new ClearCrashes.cs file is not part of the project. You need to add the file to the project in Visual Studio, it's not referenced correctly (I had to do add existing item then picked the file).

If you have it locally, click save all and commit the csproj modification which looks like

```
--- a/Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.UWP/Contoso.Forms.Puppet.UWP.csproj
+++ b/Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.UWP/Contoso.Forms.Puppet.UWP.csproj
@@ -99,6 +99,7 @@
     <Compile Include="App.xaml.cs">
       <DependentUpon>App.xaml</DependentUpon>
     </Compile>
+    <Compile Include="ClearCrashes.cs" />
     <Compile Include="EventFilterWrapper.cs" />
     <Compile Include="MainPage.xaml.cs">
       <DependentUpon>MainPage.xaml</DependentUpon>
```

Check also demo.

Reply:
Fixed in a new commit.

Reply:
Ok it works now 👍